kaggle-classification/keras_trainer/model.py

```diff
@@ -117,22 +117,33 @@ def predict(self, texts):
     data = self._prep_texts(texts)
     return self.model.predict(data)
 
-  def score_auc(self, data):
+  def score_metric(self, data, metric_name, metric_fn):
```

Comment:
Add a docstring (particularly on what is expected in a metric_fn).

Reply:
Done